docs/application-development/foundations.md

```diff
@@ -625,7 +625,21 @@ values={[{label: 'Go', value: 'go'},{label: 'Java', value: 'java'},{label: 'PHP'
 
 <TabItem value="go">
 
-Content is not available
+Use [`ConnectionOptions`](https://pkg.go.dev/go.temporal.io/sdk@v1.15.0/client#ConnectionOptions) API available in the [`go.temporal.io/sdk/client`](https://pkg.go.dev/go.temporal.io/sdk/client) package to connect a Client with TLS.
+
+```go
+	cert, err := tls.LoadX509KeyPair(clientCertPath, clientKeyPath)
```

Comment:
Eventually we'll need to snipsync this from the samples but this is fine for now.